textdb/textdb-storage/src/main/java/edu/uci/ics/textdb/storage/TableMetadata.java

```diff
@@ -0,0 +1,38 @@
+package edu.uci.ics.textdb.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.uci.ics.textdb.api.constants.JsonConstants;
+import edu.uci.ics.textdb.api.schema.Schema;
+
+public class TableMetadata {
+	private String tableName;
+	private Schema schema;
+
+	@JsonCreator
+	public TableMetadata(
+					@JsonProperty(value = JsonConstants.TABLE_NAME, required = true)
```

Comment:
I think you used tab instead of 4 spaces. Please configure your IDE to use spaces for tabs and change this.